config/puma.rb

```diff
@@ -11,6 +11,12 @@
 # cause and implement a permanent solution.
 worker_timeout 600
 
+# Set the keepalive timeout higher than what nginx is set to
+# Defaults to 20
```

Comment:
puma defaults to 20 or nginx? It's not quite clear after reading the previous line.